common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -37,7 +37,11 @@ trait Logging {
    * @param message Message to write to the log
    */
   def debug(from: AnyRef, message: String)(implicit id: TransactionId = TransactionId.unknown) = {
-    emit(DebugLevel, id, from, message)
+    if (id.meta.extraLogging) {
+      emit(InfoLevel, id, from, message)
```

Comment:
Are these reverse? 
When extraLogging is True do DebugLevel, when False to InfoLevel

Reply:
@csantanapr They are not reversed. In the case of one transaction got marked with extra logging we elevate those messages from DEBUG to INFO. As I tried to explained in the comment above, this is the only way to get them placed into Akka logging's event bus when the overall log-level is set to INFO.

Reply:
Thanks @mhenke1 for the explanation all messages will be printed with [INFO] is just when extra logging there will be more [INFO] printed not a mixed between [DEBUG] and [INFO]

Reply:
@csantanapr Exactly. For "normal" transactions messages will be send according to the log level. For transactions with "extra logging"  additionally all DEBUG messages will be send as INFO level messages.